CONTRIBUTE.md

```diff
@@ -7,49 +7,87 @@
 ### Contribution of OEO content
 Please read [OEO wiki](https://github.com/OpenEnergyPlatform/ontology/wiki/Best-Practice-Principles) carefully.
 
-### Philosophy
+### Workflow
 
 The development of a feature for this repository is inspired from the workflow described 
 by [Vincent Driessen](https://nvie.com/posts/a-successful-git-branching-model/):
 
-1. Create [an issue](https://help.github.com/en/articles/creating-an-issue) on the github repository to describe the addition to the ontology. Discussion about the implementation details should occur within this issue 
+1. Create [an issue](https://help.github.com/en/articles/creating-an-issue) on the github repository
+
+    Choose the right issue for you among the choices you have on github:
+
+    A) Including a new term to the ontology. Only 1 new term (with a few subclasses, if at all)
+    
+    B) Restructuring existing parts of the ontology
+    
+    C) Other issue
+
+    Discussion about the implementation details should occur within this issue. [Assign a project](https://help.github.com/en/github/managing-your-work-on-github/adding-issues-and-pull-requests-to-a-project-board#adding-issues-and-pull-requests-to-a-project-board-from-the-sidebar) (default "Issues") to the issue
+
 2. Create a branch from dev to work on your issue (see below, the "Branch name convention" section)
```

Comment:
See comment above. Can I do this directly after creating the issue?

Reply:
Yes